pkgs/top-level/all-packages.nix

```diff
@@ -9450,7 +9450,6 @@ in
     inherit (darwin.apple_sdk.frameworks) Security;
   };
   cargo-tarpaulin = callPackage ../development/tools/analysis/cargo-tarpaulin { };
-  cargo-tree = callPackage ../tools/package-management/cargo-tree { };
```

Comment:
Please also add a throw statement in `/pkgs/top-level/aliases.nix`

Reply:
Sure, added it.